tutorials/10-bayesian-differential-equations/10_bayesian-differential-equations.jmd

```diff
@@ -78,6 +78,12 @@ plot(sol; alpha=0.3)
 scatter!(sol.t, odedata'; color=[1 2], label="")
 ```
 
+# alternatively, we can use real-world data from Hudson’s Bay Company records (a Stan implementation can be found here: https://mc-stan.org/users/documentation/case-studies/lotka-volterra-predator-prey.html)
```

Comment:
@ChrisRackauckas would that be a good idea?

Reply:
yeah seems fine